_inc/client/my-plan/my-plan-header/checklist-cta.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { translate as __ } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+
+export default function ChecklistCta( { placeholder, siteSlug } ) {
```

Comment:
is `placeholder` prop used anywhere?

Reply:
Looks like usage of it was removed in commit 27af4f6e5. Let's clean up here, too, if it's un-used.

Reply:
👍 cf369ad324dcfdb1a56905e36f1a9a6c5f95bd0e